src/backend/CodeGenOptions.zig

```diff
@@ -18,10 +23,43 @@ pub const PicLevel = enum(u8) {
     two = 2,
 };
 
+pub const OptimizationLevel = enum {
+    unspecified,
```

Comment:
Not a fan of this existing, why not assign a default in `Driver`?

Reply:
That's only there because the default is `-O0`, which is also how we're selecting this assembly backend. I was trying to disambiguate "no optimization selected" vs `-O0` but I could also just set a separate flag if `-O0` is seen.